travelpal/assets/js/components/Profile.jsx

```diff
@@ -16,7 +72,10 @@ export default function Profile({ userInfo, friends, formOnChange,
         submitOnClick={submitOnClick}
       />
       <br />
-      <ProfileFriends userId={userInfo.id} friends={friends} />
+      {searchBtn}
```

Comment:
Can you rename "Search for Friends" to "Find Friends" and make it on the same row as the "Friends" header? And change the search label to be a `h3` and make the "View Friends" button on the same row as it as well.

Reply:
You can mimic the styling of the Travel Dates page since I have the "+ Add" button on the same row as the "Travel Dates" header, which is what I want here.

Reply:
https://github.com/kimberlypn/TravelPal/pull/29/commits/9c17ad8f73b43d2e8d067b64c0e6cdde4a4df556